package.json

```diff
@@ -14,15 +14,23 @@
     "balm": "./lib/cli.js"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "postinstall": "patch-package"
   },
   "dependencies": {
+    "@babel/core": "^7.16.7",
     "beard": "~0.9.1",
+    "browserslist-to-esbuild": "~1.1.1",
     "callsites": "3.0.0",
     "cheerio": "github:cheeriojs/cheerio#3368605edb3c5babecc8576602a9d54ccfdaef1e",
+    "esbuild": "~0.14.8",
+    "esbuild-plugin-babel": "^0.2.3",
+    "esbuild-postcss": "~0.0.4",
     "klona": "~2.0.5",
     "merge-anything": "~4.0.2",
     "normalize-selector": "0.2.0",
+    "patch-package": "^6.4.7",
+    "postcss": "^8.4.5",
     "sockette": "~2.0.6",
     "traversy": "0.0.2",
```

Comment:
With this latest approach are we still using all these packages?

Reply:
Yes, except for the patch-package stuff, which we still need to discuss for our babel implementation.